pep517/wrappers.py

```diff
@@ -110,7 +110,23 @@ def build_sdist(self, sdist_directory, config_settings):
 
     def _call_hook(self, hook_name, kwargs):
         env = os.environ.copy()
-        env['PEP517_BUILD_BACKEND'] = self.build_backend
+
+        # On Python 2, pytoml returns Unicode values (which is correct)
+        # but the environment passed to check_call needs to contain string
+        # values. We convert here by doing a default encode() which will
+        # use the system default encoding. That should do the right thing,
```

Comment:
`unicode.encode()` actually uses Python's default encoding, IIRC, not the system default. So on Python 2, it's equivalent to `.encode('ascii')`, unless someone has done the tricks necessary to call `sys.setdefaultencoding()` at startup, which is not recommended.